packages/roosterjs-editor-core/lib/darkMode/getColorNormalizedContent.ts

```diff
@@ -0,0 +1,58 @@
+export default function getColorNormalizedContent(content: string | DocumentFragment): string {
+    let el = document.createElement('div');
+    el.setAttribute('contenteditable', 'true');
```

Comment:
comment to explain contentEditable

Reply:
Or use DOMParser instead which is just a sandbox and won't execute any script

Reply:
DOMParser requires a string of HTML content, and the only way to get that from a DocumentFragment is to add it to an element, which doesn't help us.

Reply:
ok, got it.